qutip/floquet.py

```diff
@@ -707,28 +722,38 @@ def floquet_master_equation_tensor(Alist, f_energies):
         N, M = np.shape(Alist[0])
 
     Rdata_lil = scipy.sparse.lil_matrix((N * N, N * N), dtype=complex)
-    for I in range(N * N):
-        a, b = vec2mat_index(N, I)
-        for J in range(N * N):
-            c, d = vec2mat_index(N, J)
-
-            R = -1.0j * (f_energies[a] - f_energies[b])*(a == c)*(b == d)
-            Rdata_lil[I, J] = R
 
-            for A in Alist:
-                s1 = s2 = 0
-                for n in range(N):
-                    s1 += A[a, n] * (n == c) * (n == d) - A[n, a] * \
-                        (a == c) * (a == d)
-                    s2 += (A[n, a] + A[n, b]) * (a == c) * (b == d)
-
-                dR = (a == b) * s1 - 0.5 * (1 - (a == b)) * s2
-
-                if dR != 0.0:
-                    Rdata_lil[I, J] += dR
+    ###
+    # Here there were transposition errors because the paper used in
+    # the documentation has typos.
+    # NB: Works only for 1 rate matrix.
```

Comment:
Does that mean that Alist must have at most 1 element? 
Is it a new restriction? What happen if many rate matrix are given?

Reply:
I forgot to remove the comment "# NB: Works only for 1 rate matrix.", Alist can have as many elements as necessary.